tests/unit/type/list.tcl

```diff
@@ -737,6 +737,11 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
         wait_for_blocked_client
         r rpush mylist2{t} a b c
 
+        # Released on timeout.
+        $rd blmpop 0.1 1 mylist{t} left count 10
+        after 150
+        r set foo{t} bar
```

Comment:
let's change:
1. use an even shorter timeout (e.g. `0.0.1`).
2. add `$rd flush`
3. do `r ping` waiting for response to be sure redis got cpu time
4. change the `after` to 10
5. wait for 0 blocked clients after the sleep ends.

this should be reflected to the other test too.

```suggestion
        # Released on timeout.
        $rd blmpop 0.01 1 mylist{t} left count 10
        $rd flush
        r ping ;# just to make sure redis got CPU time
        after 10
        wait_for_blocked_clients_count 0
        r set foo{t} bar ;# something else to propagate after, so we can make sure the above pop didn't.
```

Reply:
good work, thanks!

Reply:
ohh, i just now realize that we can just wait for a nil response (don't need all that mess of waiting for the client to get blocked and then unblocked.
so this can be replaced with a simple `$rd read`
i now also notice that we're missing an `$rd close`, but in fact since we do `read` right after issuing the command, there's no real need for the deferring client in this case.
it can just be `r blmpop 0.01 1 mylist{t} left count 10`
or better yet: `assert_equal {} [r blmpop 0.01 1 mylist{t} left count 10]`

Reply:
i see in the last commit that you're still using `rd`, should be just `r`

Reply:
ohh. right